Week2/homework/maartjes-work.js

```diff
@@ -46,16 +46,21 @@ const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
+  const durationInHours = tasks.map(x => x.duration / 60);
+  const filterHours = durationInHours.filter(x => x >= 2);
+  for (let i = 0; i < filterHours.length; i++) {
+    filterHours[i] = filterHours[i] * hourlyRate;
+  }
   console.log(tasks, hourlyRate);
```

Comment:
You should take out all debug-type console.log statements when you submit a pull request.